tox/__init__.py

```diff
@@ -34,6 +34,22 @@ class InterpreterNotFound(Error):
 
     class InvocationError(Error):
         """ an error while invoking a script. """
+        def __init__(self, *args):
+            super(exception.Error, self).__init__(*args)
+            self.command = args[0]
+            try:
+                self.exitcode = args[1]
+            except IndexError:
+                self.exitcode = None
```

Comment:
I think this would be a little more clear as:

```python
        def __init__(self, command, exitcode=None):
            super(Error, self).__init__(command, exitcode)
            self.command = command
            self.exitcode = exitcode
```

Reply:
Thanks a lot. Just had to use `exception.Error`.

Reply:
oops yeah, sorry my sample wasn't completely copy-pastable 😆